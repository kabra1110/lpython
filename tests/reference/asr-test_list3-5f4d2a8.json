{
    "basename": "asr-test_list3-5f4d2a8",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_list3.py",
    "infile_hash": "f512b77e66fe356108135243473cf9441bc532310547070a9aeef3e9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_list3-5f4d2a8.stderr",
    "stderr_hash": "58b16800ceb54fe5e98678cf6804644236004e2e1028ef0816a59d71",
    "returncode": 1
}