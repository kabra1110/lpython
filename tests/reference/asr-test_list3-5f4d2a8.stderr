semantic error: For now pop() takes no arguments
 --> tests/errors/test_list3.py:5:9
  |
5 |     x = a.pop(2.2)
  |         ^^^^^^^^^^ 
